matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/verification/DefaultVerificationService.kt

```diff
@@ -939,9 +939,25 @@ internal class DefaultVerificationService @Inject constructor(
 
         updatePendingRequest(
                 existingRequest.copy(
-                        readyInfo = readyReq
+                    readyInfo = readyReq
                 )
         )
+
+        notifyOthersOfAcceptance(readyReq.transactionId, readyReq.fromDevice)
+    }
+
+    /**
+     * Gets a list of device ids excluding the current one.
+     */
+    private fun getOtherDeviceIds(): List<String> = cryptoStore.getUserDevices(userId)?.keys?.filter { it != deviceId }.orEmpty()
+
+    /**
+     * Notifies other devices that the current verification transaction is being handled by [fromDeviceId].
+     */
+    private fun notifyOthersOfAcceptance(transactionId: String, fromDeviceId: String) {
```

Comment:
can we have a better name for `fromDeviceId`? here it would be `acceptedFromDevice` or

Reply:
Now it's `acceptedByDeviceId`.